js/jscontent.js

```diff
@@ -0,0 +1,177 @@
+const allContent = [
+  {
+    name: 'Multi-Post Stories',
+    work: ['CANOPY', 'Back End Dev', '2015'],
+    content: `Lorem Ipsum is simply dummy text of the printing and 
+    typesetting industry. Lorem Ipsum has been the industry's standard 
+    dummy text ever since the 1500s, when an unknown printer took a galley
+     of type and scrambled it to make a type specimen book. 
+     It has survived not only five centuries, but also the leap 
+     into electronic typesetting, remaining essent`,
+    languages: ['HTML', 'css', 'javaScript'],
+  },
```

Comment:
Kindly add the `live version` and `Source code` links on each project object in the array. The links must be rendered dynamically with each modal as well. You can use any URLs at this point, they don't have to be specific to the projects but they should work.

- When I click on the `See live` button I should be redirected to the URL of the live version. You can use the live version URL of your portfolio temporarily. Then use `a` tags to attach the URLs to the `href` while mapping the project's data on each modal in this section:
```
    <div class='project'><p>See Live</p><img src='image/sm1.png'></div>
    <div class='project'><p>See Source</p><img src='image/sm2.png'></div>
```
- When I click on the `See source` button I should be redirected to the URL source code (GitHub repo). You can use the GitHub repo URL of your portfolio temporarily.